shared/js/icons_helper.js

```diff
@@ -48,7 +48,16 @@
     iconTargetSize = iconTargetSize * window.devicePixelRatio;
 
     // First look for an icon in the manifest.
-    if (siteObj.webManifestUrl && siteObj.webManifest) {
+    if (siteObj.manifest) {
```

Comment:
Yes, maybe. My concern going forward is that if we prioritise W3C manifests over Firefox manifests in general is that if both are provided there may be features supported by the Firefox manifest (like web activities) that aren't supported in the W3C manifest.

That isn't really a problem for icons though so I'd be happy to look for a web manifest first.